src/components/search-result-list/search-result-list-item/search-result-list-item-series.tsx

```diff
@@ -2,17 +2,19 @@ import React from "react";
 import { SeriesSimpleFragment } from "../../../core/dbc-gateway/generated/graphql";
 
 interface SearchResultListItemSeriesProps {
-  series: SeriesSimpleFragment;
+  series: SeriesSimpleFragment[];
 }
 
 const SearchResultListItemSeries: React.FC<SearchResultListItemSeriesProps> = ({
-  series: { part, title }
+  series
 }) => {
+  const display = series?.[0]?.numberInSeries?.display;
```

Comment:
I maybe would call this const something that is meaningful. In the data, it is called display, that's right, but I would maybe suggest calling it something like numberInSeries or stng ike that.

Reply:
👍